CONTRIBUTING.md

```diff
@@ -82,5 +82,36 @@ Each implementation should be thoroughly covered with tests. DOM structure can i
 3. Test additional interactivity / behaviour with explicit assertions
 4. Strive for 100% code coverage
 
+## Committing and Merging
+
+### Commit Messages
+OneUI's [release procedure](RELEASING.md) and [changelog](CHANGELOG.md) automation heavily relies on commit messages. Commit messages should follow the [Conventional Commit](https://www.conventionalcommits.org/en/) specifications. This is critical for the release script to determine a correct new version number. An updated changelog is generated automatically on the basis of commit messages. The commit message structure should be as follows:
+
+```
+<type>[optional scope]: <description>
```

Comment:
I thought `(optional scope)` was with round brackets?

Reply:
I've just taken that from their documentation: https://www.conventionalcommits.org/en/v1.0.0-beta.2/

But I think it kinda makes sense this way because this is to indicate it's optional. If you do use it, you should use round brackets indeed.

Reply:
Can you resolve the conversation if you're happy with this answer?